deployments/liqo/subcharts/discoveryOperator/templates/discovery-deploy.yaml

```diff
@@ -9,6 +9,148 @@ metadata:
     app: liqo.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: liqo-discovery-role
+rules:
+  - apiGroups:
+      - discovery.liqo.io
+    resources:
+      - foreignclusters
+      - searchdomains
+    verbs:
+      - get
+      - list
+      - patch
+      - update
+      - watch
+      - create
+      - delete
+  - apiGroups:
+      - discovery.liqo.io
+    resources:
+      - peeringrequests
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - config.liqo.io
+    resources:
+      - clusterconfigs
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - net.liqo.io
+    resources:
+      - networkconfigs
+      - tunnelendpoints
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - sharing.liqo.io
+    resources:
+      - advertisements
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - list
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+    verbs:
+      - get
+      - create
+      - update
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+    verbs:
+      - get
+      - create
+
+  # to satisfy ClusterRoles creation
+  - apiGroups:
+      - ""
+      - sharing.liqo.io
+    resources:
+      - advertisements
+      - advertisements/status
+      - secrets
```

Comment:
Do we need to give all of these cluster-scoped privileges for the secrets?

Reply:
Probably not, but these are the privileges needed to create the role for the remote broadcaster and the crdReplicator. That, again, probably does not need these privileges, but it is a bigger thing to do...

@palexster what do you suggest, doing it in this pr or open an issue and to do it in the future?

Reply:
I would suggest to open a dedicated issue and handle this in the very next iteration. This PR has still the merit to explicit permissions required by discovery operator, which are way too extended for the moment